base_layer/core/src/proof_of_work/sha3_pow.rs

```diff
@@ -43,7 +43,7 @@ pub fn sha3_hash(header: &BlockHeader) -> Vec<u8> {
         .chain(header.prev_hash.as_bytes())
         .chain(header.timestamp.as_u64().to_le_bytes())
         .chain(header.output_mr.as_bytes())
-        .chain(header.range_proof_mr.as_bytes())
+        .chain(header.witness_mr.as_bytes())
         .chain(header.kernel_mr.as_bytes())
```

Comment:
This needs the input_mr as well. If this is not part of the pow, it will be mutable.

Reply:
Thanks done